Resources/Prototypes/Entities/Structures/Doors/SecretDoor/secret_reinf.yml

```diff
@@ -0,0 +1,132 @@
+- type: entity
+  id: BaseSecretReinf
+  parent: BaseStructure
+  name: reinforced wall # No meta
+  suffix: secret door
+  abstract: true
+  description: Keeps the air in and the greytide out.
+  components:
+  - type: Sprite
+    sprite: Structures/Doors/secret_reinf.rsi
+    layers:
+    - state: closed
+      map: ["enum.DoorVisualLayers.Base"]
+  - type: AnimationPlayer
+  - type: Physics
+  - type: Fixtures
+    fixtures:
+      fix1:
+        shape:
+          !type:PhysShapeAabb
+          bounds: "-0.49,-0.49,0.49,0.49"
+        density: 100
+        mask:
+        - FullTileMask
+        layer:
+        - WallLayer
+  - type: Door
+    bumpOpen: false
+    clickOpen: true
+    canCrush: false
+    closeTimeOne: 0.2
+    closeTimeTwo: 0.6
+    openTimeOne: 0.6
+    openTimeTwo: 0.2
+    openingAnimationTime: 1.2
+    closingAnimationTime: 1.2
+  - type: Appearance
+  - type: Weldable
+    time: 2
+    weldedExamineMessage: null
+  - type: Airtight
+  - type: Damageable
+    damageContainer: StructuralInorganic
+    damageModifierSet: StructuralMetallic
+  - type: RCDDeconstructable
+    cost: 6
+    delay: 8
+    fx: EffectRCDDeconstruct8
+  - type: Destructible
+    thresholds:
+    - trigger:
+        !type:DamageTrigger
+        damage: 200
```

Comment:
this has the same damage thresholds as the non-reinforced secret door. if this is not intentional, it may be better to set it to be damage: 400, matching a reinforced wall